### DISCUSSION TITLE
Is there a way to override sites that use `<meta name="darkreader-lock">`?

### DISCUSSION THREAD

**Discussion by partingscientist** [darkreader/darkreader#11320] [Q&A]
I thought of removing the meta tag, but this requires disabling and re-enabling Dark Reader, something that I couldn't figure out how to do programmatically. Are there any other alternatives? Did I miss something? Thank you.

**Comment by alexconstant9108** [comment]
Ideally, it would be an option in Dark Reader. Something like "Override site specific darkreader-lock meta tags"

**Comment by listvin** [comment]
That war will never end, then devs would ask for tag to override user override :laughing:

Tho I believe _regarding UX_ it can be done by turning button "Enabled for current website" to "Force for current website". Now this button does just nothing when `darkreader-lock` meta tag is present. Not sure how much collateral complexity it can bring.
